sio-digital-team/active-safe
# Sports Overview grouped in ACF and SLP template updated to reflect that

https://trello.com/c/2p2R2rN2


**Changes:**

- [x] Group fields in ACF to sit under one tab
- [x] Update the front end for the new object
  

**How to test:**
Use the updated layout to add content to the intro section of the sports landing page and see if its still working as intended like [Staging](http://active-safe.siolab.ca/)
